src/tools/cephfs/Resetter.cc

```diff
@@ -55,6 +63,13 @@ void Resetter::reset(int rank)
     if (r == -ENOENT) {
       cerr << "journal does not exist on-disk. Did you set a bad rank?"
```

Comment:
It looks to me like users can still specify the wrong rank and then they get this hard reset behavior. I think we should 
1) try and check the rank's validity (by grabbing the mdsmap?),
2) only do a hard reset if they pass in a flag for it.